### ISSUE TITLE
Feature Request: Calculate "amount" number

### ISSUE THREAD

**Issue by michaelzero00** [Jon-Becker/nft-generator-py#30]
Right now it's a bit of trial and error trying to get the right amount files to generate based on the input layers.

It would be great to calculate this figure based on the config.

**Comment by michaelzero00** [comment]
But that only works when all traits are set to 100% rarity right?

**Comment by Jon-Becker** [comment]
No. If there's one layer with 2 possibilities with weights [99, 1], there's still 2 possible combinations and the program will run until it gets both if editions is set to 2.
